integration-tests/plugin-tests/src/test/kotlin/com/r3/conclave/integrationtests/general/tests/plugin/CreateDummyKeyTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.r3.conclave.integrationtests.general.tests.plugin
+
+import com.r3.conclave.integrationtests.general.commontest.TestUtils.calculateMrsigner
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.nio.file.Path
+import kotlin.io.path.deleteIfExists
+
+class CreateDummyKeyTest : AbstractPluginTaskTest() {
+    override val taskName: String get() = "createDummyKey"
```

Comment:
Isn't a bit more clear if we pass the task name in the `runTask` function?

Reply:
I don't think so. Isn't it clear that this class is testing `createDummyKey`, so any reference to `runTask` must refer to that?

Reply:
This was the first class I reviewed, and it was not immediately clear that we were running that task. As a first time reader of the class, I immediately jumped to the @Test section and `runTask` does not tell much.
Now that I understand the mechanism, things are a bit clearer.